.github/workflows/tests.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     timeout-minutes: 30
     env:
       MPLBACKEND: agg
-      PIP_ARGS: --upgrade --use-feature=2020-resolver -e
```

Comment:
Can you explain to me a bit more what this change means?

Reply:
The `--use-feature=2020-resolver` option is leading to a failure of the workflow. As far as I can see it is deprecated and as the workflows in other hyperspy-related projects don't use it, I think we can do without. I guess, it comes from some template that you used when setting up the tests.